src/Core/Worker.php

```diff
@@ -406,7 +400,7 @@ private static function execFunction($queue, $funcname, $argv, $method_call)
 		$rest    = number_format(max(0, $up_duration - (self::$db_duration + self::$lock_duration)), 4);
 		$exec    = number_format($duration, 4);
 
-		Logger::info('Performance:', ['total' => $dbtotal, 'count' => $dbcount, 'stat' => $dbstat, 'write' => $dbwrite, 'block' => $dblock, 'rest' => $rest, 'exec' => $exec]);
+		$workerLogger->info('Performance log.', ['total' => $dbtotal, 'count' => $dbcount, 'stat' => $dbstat, 'write' => $dbwrite, 'block' => $dblock, 'rest' => $rest, 'exec' => $exec]);
```

Comment:
Why is ```$workerLogger->``` used here?

Reply:
hm... you're right, I'll undo it

Reply:
The question - of course - is directed to all comparable calls there.

Reply:
OK, maybe I was a little bit quick with the undo.. I thought that the performance - or any similar - logs are for one worker-execution. So it would be good to add the worker information to this kind of logs either.

The question is, if it's helpful for you to profile worker start/stops or if it doesn't matter?

Reply:
The worker id is used to trace down bugs in some worker task. All the actions that are done in the worker class itself are not important for this debugging. The logging here is important for measuring performance or to trace bugs in the worker processing itself.

I think that it is enough to have a single logger call of this kind before the execution of the task and a single logger call at the end. So we can see the start and we see if it ended correctly.